README.md

```diff
@@ -1,12 +1,37 @@
 [![License: LGPL v3](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0)
-# BHoM
-
-A great place to start is reading our Wiki [here](https://github.com/BHoM/documentation/wiki) including pages like the [Structure of the BHoM](https://github.com/BHoM/documentation/wiki/Structure-of-the-BHoM) and [Using the BHoM](https://github.com/BHoM/documentation/wiki/Using-the-BHoM).
-
-## Quick start ##
-
-Try the [installer](http://bhom.xyz/assets/installers/v2.1/BHoM%20Alpha%20v2.1.0.5%20Installer.exe) and a selection of [sample scripts](https://github.com/BHoM/samples).
-
+## Install me
+To install the Python_Toolkit:
+1. Compile:  
+    1. [BHoM](https://github.com/BHoM/BHoM)  
+    1. [BHoM_Engine](https://github.com/BHoM/BHoM_Engine)  
+    1. [BHoM_Adapter](https://github.com/BHoM/BHoM_Adapter)   
+    1. [BHoM_UI](https://github.com/BHoM/BHoM_UI)  
+	  1. [Rhinoceros_Toolkit](https://github.com/BHoM/Rhinoceros_Toolkit)  
+	  1. [Grasshopper_Toolkit](https://github.com/BHoM/Grasshopper_Toolkit)  
+	  1. Python_Toolkit (this repo)
+1. Open ai UI of your choice (e.g. Grasshopper)
+1. Run the `InstallPythonToolkit` component and wait for the installation to finish.
+1. he installation has succeeded if the packages installed are:
+	  - Python 3.7
+  	- jupyter
+  	- matplotlib
+  	- Python_Toolkit
+1. Restart your UI
+
+To check that all went ok you can run the component `BH.Engine.Python.Compute.Import` with input a string `Python_Toolkit`,
```

Comment:
To check whether all went well you can run the component `BH.Engine.Python.Compute.Import` with a input of a string as `Python_Toolkit`,

Reply:
```suggestion
To check whether all went well you can run the component `BH.Engine.Python.Compute.Import` with a input of a string as `Python_Toolkit`,
```